rocketpy/AeroSurfaces.py

```diff
@@ -234,20 +253,20 @@ def evaluateLiftCoefficient(self):
 
         Returns
         -------
-        self.cl : Function
-            Function of the angle of attack (Alpha) and the mach number
-            (Mach) expressing the lift coefficient of the nose cone. The inputs
-            are the angle of attack (in radians) and the mach number.
-            The output is the lift coefficient of the nose cone.
+        None
         """
         # Calculate clalpha
-        self.clalpha = 2 * self.radiusRatio**2
+        self.clalpha = Function(
```

Comment:
I can understand the reason why the mach is being ignored in the first self.clalpha definition, but please keep in mind that a regular user/reader won't understand it so easily. 

Matbe adding a comment or TODO could make future lives better